code/datums/keybinding/carbon.dm

```diff
@@ -65,3 +65,27 @@
 /datum/keybinding/carbon/select_harm_intent/down(client/user)
 	user.mob?.a_intent_change(INTENT_HARM)
 	return TRUE
+
+/datum/keybinding/mob/specialclick
+	key = "Control"
```

Comment:
If `ctrl` is used, then the interactions with `CtrlClick` will need to be addressed.